src/components/interaction-designer/resource-editors/ResourceEditor.vue

```diff
@@ -175,22 +175,21 @@ import {cloneDeep} from "lodash";
       this.resource_setOrCreateValueModeSpecific({
         resourceId: this.resource.uuid,
         filter: {languageId: langId, contentType: SupportedContentType.AUDIO, modes: [SupportedMode.IVR]},
-        value: `https://www.viamo.io/audiofiles/play/${filename}/${extension}`
+        value: filename,
```

Comment:
Hi @safydy-r, I'm a bit confused here. In our stubbed `/audiofiles/upload` devServer endpoint configuration we don't respond with a `filename` property, only a `description` set to `req.query.flowFilename`. 

What's your recommendation here? Was there a `filename` on the original implementation? Should we add `filename` to our stubbed endpoint?

Reply:
@bzabos , in the route stub `audiofiles/upload` devServer, I just imitated voto5's response as commented here : https://github.com/FLOIP/flow-builder/blob/VMO-3167-add-upload-audio-resource-for-audio-library/vue.config.js#L19

It came from this backend code : https://bitbucket.org/voto/voto5-core-lib/src/2ecf941b1f3881c8d7a7440dd1c940436fd75640/src/AudioFiles/AudioFile.php#lines-326

There was no `filename` property. 

To my point of view, it is always good to have every metadata related to the uploaded file, especially the filename. I would suggest we add it. But this will require we edit the backend code I mentioned above when we integrate into voto5.  I'm happy add it to our stubbed endpoint but need your confirmation then.

Reply:
Note: after chat, we will just use `value: description`